app/src/main/java/ca/klapstein/baudit/data/Email.java

```diff
@@ -39,11 +39,21 @@ static public boolean isValid(String email) {
         return emailValidator.isValid(email);
     }
 
+    /**
+     * Get the {@code String} of the {@code Email}
```

Comment:
I would be more descriptive here.

something like `Get the {@code String} representation of the {@code Email}.`